test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -803,6 +886,23 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
         $response.Content.UserAgent | Should Match $Pattern
     }
 
+    It "Verifies Invoke-WebRequest Certificate Authentication Fails without -Certificate"{
+        $command = 'Invoke-WebRequest https://prod.idrix.eu/secure/'
```

Comment:
@markekraus - I don't have a quick solution to mocking the call; however, I still question the use of the link (https://prod.idrix.eu/secure/).  I found a reference to it on stackoverflow but I cannot find any information on the parent site indicating its for public use.  Have I missed something?